go/cmd/osmosis/main.go

```diff
@@ -47,23 +46,31 @@ func main() {
 		}
 	}
 
-	encoding := cosmos.NewEncoding(gammtypes.RegisterInterfaces, lockuptypes.RegisterInterfaces)
+	encoding := cosmos.NewEncoding()
```

Comment:
We will want to register these osmosis specific types for tx decoding

Reply:
done